README.md

```diff
@@ -1 +1,682 @@
-# messaging-with-activemq
\ No newline at end of file
+# messaging-with-activemq
```

Comment:
You need to include travis test integration

Reply:
Travis test has been integrated and tests are passing.